resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java

```diff
@@ -111,6 +111,17 @@ public void getRidStatusTest() throws ApisResourceAccessException {
 		responseWrapper.setResponse(list);
 		residentService.getRidStatus(requestDTO);
 
+		// UIN_GENERATED
+		response.setStatusCode("UIN_GENERATED");
+		responseWrapper.setResponse(list);
+		residentService.getRidStatus(requestDTO);
```

Comment:
Why this is no assert statements ?

Reply:
yes sir i have added now in this commit. https://github.com/mosip/resident-services/pull/227/commits/aedb823be1437fa4ed2061104b57eec75ad1ef4f

Reply:
Why residentService.getRidStatus(requestDTO) is called twice ?

Reply:
Yes sir for each validation response like UIN_GENERATED and AWAITING_INFORMATION separate variable required to test different assert conditions.